provisioner/template.go

```diff
@@ -581,6 +582,26 @@ func nodeCIDRMaxPods(maskSize int64, extraCapacity int64) (int64, error) {
 	return maxPods, nil
 }
 
+// addressNFromIPV6CIDR takes an IPv6 CIDR and returns the Nth address in the
+// subnet.
+func addressNFromIPV6CIDR(cidr string, n int) (string, error) {
+	_, ipNet, err := net.ParseCIDR(cidr)
+	if err != nil {
+		return "", err
+	}
+
+	ip := ipNet.IP
+	ip = ip.To16()
+	if ip == nil {
+		return "", fmt.Errorf("invalid IP address: %s", ipNet.IP)
+	}
+	ip = ip.Mask(ipNet.Mask)
+	for i := 0; i < n; i++ {
+		ip[15]++
+	}
+	return ip.String(), nil
+}
+
```

Comment:
We should definitely cover this with a unit test

Reply:
done